src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/DataController.cs

```diff
@@ -140,6 +143,20 @@ public async Task<IActionResult> Get(int instanceOwnerPartyId, Guid instanceGuid
             // check if dataId exists in instance
             if (dataElement != null)
             {
+                string orgFromClaim = User.GetOrg();
+
+                if (!string.IsNullOrEmpty(orgFromClaim))
+                {
+                    _logger.LogInformation($"App owner download of {instance.Id}/data/{dataGuid}, {instance.AppId} for {orgFromClaim}");
+
+                    // update downloaded structure on data element
+                    dataElement.AppOwner ??= new ApplicationOwnerDataState();
+                    dataElement.AppOwner.Downloaded ??= new List<DateTime>();
+                    dataElement.AppOwner.Downloaded.Add(DateTime.UtcNow);
+
+                    _ = await _dataRepository.Update(dataElement);
```

Comment:
If we want to suppress any exceptions from the update call, _ = is ok to use. However if we are interested in that exception, do not use _ = and just write:
`await _dataRepository.Update(dataElement);`

Reply:
@dskogan

Reply:
We should take exceptions into considerations. så Await

Reply:
Does it look right now ?